roles/nginx/files/nginx.conf

```diff
@@ -21,7 +21,7 @@ http {
   access_log /var/log/nginx/access.log main;
 
   upstream application {
-    server unix:///var/pyapps/hhs_o_server/pyapps.sock;
+    server unix:///var/pyapps/hhs_o_server/run/pyapps.sock;
```

Comment:
why the move to `/run`?

Reply:
SELinux was preventing NGINX from creating the pyapps.sock file. The directory was needing to be labeled with the type httpd_sys_rw_content_t for write access.  So I created the /run sub directory to have that type, without affecting other files/usage in the parent dir.  I believe that NGINX only needs write access for the .id and .sock files from the logging information so far.

Reply:
do these need to be updated too?

https://github.com/CMSgov/bluebutton-web-deployment/blob/master/roles/nginx/files/uwsgi.ini#L6
https://github.com/CMSgov/bluebutton-web-deployment/blob/master/roles/nginx/tasks/main.yml#L40